components/dashboard/src/settings/PersonalAccessTokens.tsx

```diff
@@ -205,6 +206,8 @@ export function PersonalAccessTokenCreateView() {
                 if (!tokenId) {
                     return;
                 }
+                // update UI to `edit style` immediately
+                setEditTokenID(tokenId);
```

Comment:
I removed because I was planning to add loading widget too

Reply:
[pushed](https://github.com/gitpod-io/gitpod/pull/15015/commits/08e54ef9fd24079413df623a984f3b4bb23e457a) to add loading state